sycl/plugins/esimd_emulator/pi_esimd_emulator.cpp

```diff
@@ -1531,15 +1551,53 @@ pi_result piEnqueueMemBufferFill(pi_queue, pi_mem, const void *, size_t, size_t,
   DIE_NO_IMPLEMENTATION;
 }
 
-pi_result piEnqueueMemBufferMap(pi_queue, pi_mem, pi_bool, pi_map_flags, size_t,
-                                size_t, pi_uint32, const pi_event *, pi_event *,
-                                void **) {
-  DIE_NO_IMPLEMENTATION;
+pi_result piEnqueueMemBufferMap(pi_queue Queue, pi_mem Buffer,
```

Comment:
for future: please add comment why no real mapping occurs here and this is OK

Reply:
Done.